contrib/experimental/great_expectations_experimental/tests/rule_based_profiler/data_assistant/test_growth_numeric_data_assistant.py

```diff
@@ -413,9 +413,9 @@ def test_spark_happy_path_growth_numeric_data_assistant(
     4. Configuring BatchRequest to load 2020 January data
     5. Configuring and running Checkpoint using BatchRequest for 2020-01, and 'taxi_data_2019_suite'.
     """
-    from pyspark.sql.types import StructType
```

Comment:
Why not import StructType here from `great_expectations.compatibility.pyspark`?

Reply:
@anthonyburdi The style for this was in the name of generality -- covers everything under `types.`.

Reply:
Wouldn't this raise an `AttributeError` though if pyspark isn't installed and `sparktypes` is `NotImported` since there is no `StructType` on `NotImported`?

Reply:
@anthonyburdi It seems that it potentially could -- you are right, but it has not happened in our tests.  I need to check to see if the code involved when we use `sparktypes` is in general executed only when `pyspark` is imported and exists (then we should be fine).  More soon.  Thanks.

Reply:
@anthonyburdi I did a thorough check -- in all ~26 files where we use `sparktypes`, it is within methods/modules where a valid `pyspark` installation is expected, so we should be good here.  I just added a couple of tiny cosmetic changes (a type hint; a couple of commented imports for readability), based on my observations while checking these files.  Thanks!